src/lib/src/select-container.component.ts

```diff
@@ -89,7 +90,7 @@ export class SelectContainerComponent implements AfterViewInit, OnDestroy {
   @Output() select = new EventEmitter<any>();
 
   private _tmpItems = new Map<SelectItemDirective, Action>();
-  private _selectedItems: Array<any>;
+  private _selectedItems$ = new BehaviorSubject<Array<any>>([]);
```

Comment:
In general I like the idea of tackling this with a `BehaviorSubject` and I feel this could even be more lightweight than the Proxy solution. Plus I get your reasoning that you have to support IE11 because Microsoft will continue the support for it until Windows 7's support is dropped.